.github/workflows/docker-tag.yml

```diff
@@ -31,5 +31,16 @@ jobs:
         git push --set-upstream origin rs-automatic-updates/$newtag
         echo ${{ secrets.JENKINS_UPDATE }} | gh auth login --with-token
         gh pr create -R datakaveri/iudx-deployment --base master --fill 
+        git remote add upstream https://github.com/datakaveri/iudx-deployment.git
+        git fetch upstream 4.0.0
+        git checkout 4.0.0
+        git checkout -b rs-4.0.0-automatic-updates/$testtag
+        sed -i s/$oldtag/$newtag/g Docker-Swarm-deployment/single-node/resource-server/rs-stack.yaml
+        sed -i s/$oldappversion/$newappversion/g K8s-deployment/Charts/resource-server/Chart.yaml
+        sed -i s/$oldtag/$newtag/g K8s-deployment/Charts/resource-server/values.yaml
+        git add Docker-Swarm-deployment/single-node/resource-server/rs-stack.yaml K8s-deployment/Charts/resource-server/values.yaml K8s-deployment/Charts/resource-server/Chart.yaml
+        git commit --allow-empty -m "updated rs docker image tag to $newtag"
+        git push --set-upstream origin rs-4.0.0-automatic-updates/$testtag
```

Comment:
it should be 
``git push --set-upstream origin rs-4.0.0-automatic-updates/$newttag``